root/report/index.tt

```diff
@@ -29,6 +29,14 @@
             <li><a href="[% c.uri_for_action('/report/show', 'ArtistsDisambiguationSameName') %]">[% l('Artists with disambiguation the same as the name') %]</a></li>
         </ul>
 
+        [% IF c.user.is_account_admin -%]
```

Comment:
Other than not outputting the link here, where does it restrict access to account admins?

Reply:
Nowhere else. Would you prefer it to be done in https://github.com/metabrainz/musicbrainz-server/pull/556/files#diff-068aefca9b2fdb274a6a5cd3cb07a5e2 ?

Reply:
Oops, fixed in https://github.com/metabrainz/musicbrainz-server/pull/556/commits/cc1a4fdc419081b809f511d471435208a00b0d5e#diff-c720a88e431a4eb472afdd86f5d7206fR12